src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -100,10 +91,7 @@ public void deletePrimaryUserAffiliation(String eventPayload) {
       PrimaryAffiliationEvent affiliationEvent = createPrimaryAffiliationEvent(userEvent);
       String data = objectMapper.writeValueAsString(affiliationEvent);
 
-      // context is changed in deleteShadowUsers() method and context is empty after deleteShadowUsers() method, so we need to set context again.
-      try (var context = new FolioExecutionContextSetter(prepareContextForTenant(centralTenantId, folioModuleMetadata, currentContext))) {
-        kafkaService.send(KafkaService.Topic.CONSORTIUM_PRIMARY_AFFILIATION_DELETED, consortiaTenant.getConsortiumId().toString(), data);
```

Comment:
after resent changes in folio-spring-base we doesn't need to set back again context in method. it automatically back to initial context in scope of method.

Reply:
I tested manually I didn't see any errors